### ISSUE TITLE
Other packages to add

### ISSUE THREAD

**Issue by Xeoncross** [Xeoncross/go-cache-benchmark#1]
There are other Go libraries to add and benchmark as well

- https://github.com/karlseguin/ccache
- https://github.com/VictoriaMetrics/fastcache

**Comment by tarndt** [comment]
I would have liked to see https://github.com/valyala/ybc in the comparison. In the past I have found it outperforms most Go memory-based impls and it can mmap SSD based storage.

**Comment by yogesh-desai** [comment]
Hi @Xeoncross 
Do you accept PR?
I wanted to add 2 more caches,

- hashicorp "github.com/hashicorp/golang-lru"

- koding "github.com/koding/cache"

Anyways, if you don't then, below is the related code for it. :)
   
 
    func BenchmarkKodingCache(b *testing.B) {
		c := koding.NewMemoryWithTTL(time.Duration(60) * time.Second)
	
		b.Run("Set", func(b *testing.B) {
			for i := 0; i < b.N; i++ {
				c.Set(strconv.FormatInt(int64(i), 10), i)
			}
		})
	
		b.Run("Get", func(b *testing.B) {
			for i := 0; i < b.N; i++ {
			
				value, err := c.Get(strconv.FormatInt(int64(i), 10))
				if err == nil {
					_ = value
				}
			}
		})
	}
	
	func BenchmarkHashicorpLRU(b *testing.B) {
		// c := cache2go.Cache("test")
		c, _ := hashicorp.New(10)
	
		b.Run("Set", func(b *testing.B) {
			for i := 0; i < b.N; i++ {
				c.Add(i, i)
			}
		})
	
		b.Run("Get", func(b *testing.B) {
			for i := 0; i < b.N; i++ {
			
				value, err := c.Get(i)
				if err == true {
					_ = value
				}
			}
		})
	}
`

**Comment by Xeoncross** [comment]
@yogesh-desai yes, you're welcome to add these as a PR or when I have time I'll them myself.

I avoided https://github.com/hashicorp/golang-lru in the comparison because it does not have TTL / expiration times for the objects in the collection. It is only a fixed-size LRU cache. Still, I'm open to adding it for comparison.

**Comment by yogesh-desai** [comment]
Yes, you are right. For my use case, I just added TTL feature in https://github.com/hashicorp/golang-lru
which again helps a lot and I'm sure it will perform very well too.
Thank you @Xeoncross , will raise a PR soon.

**Comment by alexon1234** [comment]
Hi, 
Do you have in mind to add [ristretto](https://github.com/dgraph-io/ristretto)? 
Thanks

**Comment by Xeoncross** [comment]
Yes, ristretto just came out and it's probably what is a majority of cases should use.

**Comment by alexon1234** [comment]
@Xeoncross if you have a second, could you update the readme with the ristretto benchmark? 
Thanks :D

**Comment by Xeoncross** [comment]
Sure, updated. Thanks again!

**Comment by tarndt** [comment]
@Xeoncross Any progress on original target of this issue https://github.com/VictoriaMetrics/fastcache ?
